src/utils/radiotap_parser.cpp

```diff
@@ -132,6 +132,7 @@ struct RadioTapFlags {
 #endif
 
 void align_buffer(const uint8_t* buffer_start, const uint8_t*& buffer, uint32_t size, size_t n) {
+    (void)size;
```

Comment:
This is a local function so I'd rather just remove the `size` parameter rather than ignoring it. It's only called once in this file so that should be straightforward.